dioptas/tests/controller_tests/test_BatchController_part1.py

```diff
@@ -175,11 +175,11 @@ def test_change_scale(batch_controller, batch_widget, batch_model, load_proc_dat
 
     batch_controller.change_scale_log(MockMouseEvent())
     assert batch_controller.scale == np.log10
-    assert np.all(batch_widget.stack_plot_widget.img_view.img_data == 2.)
+    assert np.allclose(batch_widget.stack_plot_widget.img_view.img_data, 2.)
```

Comment:
On CI machines the exact comparison fails (not reproduced locally).
Fix `test_change_scale` error:

```
batch_controller = <dioptas.controller.integration.BatchController.BatchController object at 0x7f91ca8471c0>
batch_widget = <dioptas.widgets.integration.BatchWidget.BatchWidget object at 0x7f91c99555a0>
batch_model = <dioptas.model.BatchModel.BatchModel object at 0x7f91bef98f70>
load_proc_data = ['/home/runner/work/Dioptas/Dioptas/dioptas/tests/controller_tests/../data/lambda/testasapo1_1009_00002_m1_part00000.n...home/runner/work/Dioptas/Dioptas/dioptas/tests/controller_tests/../data/lambda/testasapo1_1009_00002_m1_part00004.nxs']

    def test_change_scale(batch_controller, batch_widget, batch_model, load_proc_data):
        batch_widget.activate_stack_plot()
        batch_model.data[:, :] = 100
    
        batch_controller.change_scale_log(MockMouseEvent())
        assert batch_controller.scale == np.log10
>       assert np.all(batch_widget.stack_plot_widget.img_view.img_data == 2.)
E       assert False
E        +  where False = <function all at 0x7f91e4100c70>(array([[2.0000002, 2.0000002, 2.0000002, ..., 2.0000002, 2.0000002,\n        2.0000002]], dtype=float32) == 2.0)
E        +    where <function all at 0x7f91e4100c70> = np.all
E        +    and   array([[2.0000002, 2.0000002, 2.0000002, ..., 2.0000002, 2.0000002,\n        2.0000002]], dtype=float32) = <dioptas.widgets.plot_widgets.ImgWidget.IntegrationBatchWidget object at 0x7f91c9956e60>.img_data
E        +      where <dioptas.widgets.plot_widgets.ImgWidget.IntegrationBatchWidget object at 0x7f91c9956e60> = <dioptas.widgets.integration.BatchWidget.BatchStackWidget object at 0x7f91c9956830>.img_view
E        +        where <dioptas.widgets.integration.BatchWidget.BatchStackWidget object at 0x7f91c9956830> = <dioptas.widgets.integration.BatchWidget.BatchWidget object at 0x7f91c99555a0>.stack_plot_widget

dioptas/tests/controller_tests/test_BatchController_part1.py:178: AssertionError
```